packages/cf-component-card/src/CardToolbarLink.js

```diff
@@ -1,6 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Link from 'cf-component-link';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ isActive }) => ({
+  position: 'relative',
+  display: 'inline-block',
+  padding: '0.73333rem 2em 0.73333rem 0.5rem',
+
+  '&::after': {
+    content: "'\\f003'",
+    fontFamily: '"cloudflare-font"',
+    fontStyle: 'normal',
+    fontWeight: 'normal',
+    marginTop: '-0.7em',
+
+    position: 'absolute',
+    top: '60%',
```

Comment:
Yup! Changing it to `60%` fixed the alignment issues we currently have on production between the text and the arrow icon.

![image](https://user-images.githubusercontent.com/318208/28071632-68effb0a-6648-11e7-8b4d-96225829d023.png)